src/pages/landing/deriv-go/_footer-banner.js

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import styled from 'styled-components'
+import { graphql, useStaticQuery } from 'gatsby'
+// import { Text } from 'components/elements/typography'
+import { Flex, Show } from 'components/containers'
+// import { localize } from 'components/localization'
+import device from 'themes/device.js'
+import DerivSmallLogo from 'images/svg/deriv-go/dgo-logo-footer.svg'
+import { Header, LocalizedLinkText, QueryImage } from 'components/elements'
+import { localize, Localize } from 'components/localization'
+import { LinkButton } from 'components/form'
+import { deriv_go_playstore_url } from 'common/utility'
+
+const query = graphql`
+    query {
+        deriv_go_qr_code: file(relativePath: { eq: "deriv-go/qr-code.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const BannerWrapper = styled(Flex)`
+    overflow: hidden;
+    background-color: var(--color-grey-30);
+    padding: 40px 0;
+    @media ${device.laptopM} {
+        padding: 20px;
+    }
+    @media ${device.tabletL} {
+        padding: 20px 16px;
+    }
+`
+const MainWrapper = styled(Flex)`
+    max-width: 996px;
+    margin: 0 auto;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    @media ${device.tabletL} {
+        max-width: 58.8rem;
+    }
+`
+const DGoSmallLogo = styled.img`
+    width: 120px;
+    height: 120px;
+    margin-right: 24px;
+    @media ${device.tabletL} {
+        width: 32px;
+        height: 32px;
+        margin-right: 8px;
+    }
+`
+const TextWrapper = styled(Flex)`
+    margin-right: 24px;
+    flex-direction: column;
+`
+const StyledHeader = styled(Header)`
+    margin-bottom: 8px;
+    line-height: 40px;
+    @media ${device.tabletL} {
+        font-size: 16px;
+        line-height: 24px;
+        margin-bottom: 0;
+    }
+`
+const StyledHeaderSmall = styled(Header)`
+    margin-bottom: 8px;
+    line-height: 36px;
+    @media ${device.tabletL} {
+        font-size: 14px;
+        line-height: 21px;
+        margin-bottom: 0;
+    }
+`
+const StyledHeaderSub = styled(Header)`
+    line-height: 1.5;
+`
+// const DesktopImageWrapper = styled(Container)`
+//     width: 100%;
+//     @media ${device.tabletL} {
+//         display: none;
+//     }
+// `
+// const MobileImageWrapper = styled(Container)`
+//     display: none;
+//     @media ${device.tabletL} {
+//         display: flex;
+//         width: 100%;
+//     }
+// `
+const QRCodeImgWrapper = styled(Flex)`
+    width: 96px;
+    height: 96px;
+    align-items: center;
+`
+const BtnDiv = styled(Flex)`
```

Comment:
Maybe `ButtonWrapper` might be a bit more descriptive for future developers here.

Reply:
Amended in the latest commit